requirements.txt

```diff
@@ -6,3 +6,4 @@ youtube-dl>=2019.4.17
 jupyterlab
 opencv-python>=3.3.0.10
 pillow
+pyunpack
```

Comment:
This dependency is only used by API, requirements.txt is for DeOldify. This dependency is not required to run notebooks or Colab, I believe it should be moved to requirements-api.txt. Perhaps there are more dependencies that should be moved.